.github/workflows/test-images.yaml

```diff
@@ -1,14 +1,14 @@
 name: Create Test Images
 on:
   push:
-    branches:
-      - "staging"
     paths:
       - 'test/e2e/fixtures/Dockerfile.*'
+  workflow_dispatch:
 
 jobs:
   list-dockerfiles:
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
Is this if needed? I think you want to run this under all circumstances?

Reply:
without that condition the workflow works fine 
I tested in my forked repo  - [Action](https://github.com/sudharshanibm3/cloud-api-adaptor/actions/runs/5282422628)
<img width="1062" alt="Screenshot 2023-06-16 at 12 08 43 AM" src="https://github.com/confidential-containers/cloud-api-adaptor/assets/129826358/35514952-5a43-4a95-9eec-8f2bf72fc243">